### PR TITLE
readme: reorganization and additional instructions

### PR DESCRIPTION
**Pull request by laurenkrugen-navapbc**
## 🎫 Ticket

https://jira.cms.gov/browse/BCDA-xxx

## 🛠 Changes

Restructured the readme to follow logical flow of first time setup, clarified some sections like prereqs for running tests, and added a section on how to run and interact with the api locally.

## ℹ️ Context for reviewers

general improvements to readme.

## ✅ Acceptance Validation

--

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,16 @@
 
 [![Build Status](https://github.com/CMSgov/bcda-app/actions/workflows/ci-workflow.yml/badge.svg?branch=master)](https://github.com/CMSgov/bcda-app/actions?query=branch%3Amaster)
 
-## Dependencies
+## **Documentation**
+
+[API documentation](https://bcda.cms.gov/sandbox/user-guide/)
+
+#
```
Comment (kyeah) [inline README.md:9]: is this blank header intentional? 🙂

File: README.md [hunk 2]
```diff
@@ -14,100 +23,160 @@ For further ansible documentation see: (https://docs.ansible.com/ansible/2.4/vau
...
+
+After we successfully retrieve a token, we can make a request to any of the available endpoints. The PostMan collections under `test/postman_test/...` can be imported into postman directly and used to make requests, or you can use any tool like `curl`.
+
+#
+
+## **Run Tests**
 
-Run tests and produce test metrics.
-The items identified above in the `Build/Start` section are prerequisites to running tests.
+**Prerequisite**: Before running the tests and produce test metrics, you must complete the `Build Images` step from `Start the API` section.
```
Comment (kyeah) [inline README.md:154]:
```suggestion
**Prerequisite**: Before running the tests and producing test metrics, you must complete the `Build Images` step from `Start the API` section. 
```

File: README.md [hunk 3]
```diff
@@ -14,100 +23,160 @@ For further ansible documentation see: (https://docs.ansible.com/ansible/2.4/vau
...
+*Note: If this is the first time running the tests follow instructions in the 'Running unit tests locally' section of this README. Then run:
+```sh
+make load-fixtures
+```
 
 In order to keep the test feedback loop optimized, the following items must be handled by the caller (and are not handled by the test targets):
 
 - Ensuring the compose stack is up and running
 - Ensuring the database has been seeded
 - Managing images/containers (if Dockerfile changes have occurred, an image rebuild is required and won't occur as part of the test targets)
```
Comment (kyeah) [inline README.md:177]: I think these steps are only for running tests through VSCode / using `go` directly (i.e. not using make unit-test). Could we clarify that here somehow? 🤔 I've generally run `make unit-test` directly which automatically runs `make-unit-test-db` and such without me having to do anything 😄 

Thanks!

**Review by kyeah — DISMISSED** [review]
Thanks! looks good overall, adding some minor comments :)

File: README.md (hunk 1 above)
Comment (laurenkrugen-navapbc) [reply README.md:9]: oh that does look odd -- I didn't notice that previously. I'll remove that!

File: README.md (hunk 3 above)
Comment (laurenkrugen-navapbc) [reply README.md:176]: nice, I didn't catch that previously! I'll update that as well.

**Review by kyeah — APPROVED** [review]
🎉

**Pull request merged**